migrations/2018-05-21-00-member-usernames.ts

```diff
@@ -2,13 +2,17 @@
  * This is an example script for how to migrate firestore data, in this case
  * how we did the migration from member usernames with $ to using .
  */
+
+import * as path from "path";
 import * as admin from "firebase-admin";
 
 async function migrateMemberUsernamesDollarToDot(
   pathToFbKey: string,
   projectId: string
 ): Promise<void> {
-  const firebaseKey = require(pathToFbKey);
+  const firebaseKey = require(path.resolve(
+    path.join(process.cwd(), pathToFbKey)
```

Comment:
Probably ought to use __dirname to use the location of the file instead of cwd, which is dependent on where node was invoked